src/elm/datetime.coffee

```diff
@@ -8,12 +8,6 @@ module.exports.DateTime = class DateTime extends Expression
   constructor: (@json) ->
     super
 
-  # Define a simple getter to allow type-checking of this class without instanceof
-  # and in a way that survives minification (as opposed to checking constructor.name)
-  Object.defineProperties @prototype,
-    isDateTime:
-      get: -> true
```

Comment:
This wasn't removed but rather it is on the src/datatype/datetime.coffee instead of both the src/elm/datetime.coffee and the src/datatype/datetime.coffee.

Since the src/elm/datetime.coffee returns the src/datatype/datetime.coffee, this was redundant.

Reply:
Responded too quickly before I really read your whole comment.

I don't think there was any reason to have this in both places--since the constructor here returns the datatype/datetime, the object has the `isDateTime` property already.